Allow Viewing of Disabled Params (#902)
algorand/indexer: README.md

~$ curl localhost:8980/transactions -H "X-Indexer-API-Token: your-token"
```

## Disabling Parameters 

The Indexer has the ability to selectively enable or disable parameters for endpoints.  Disabling a "required" parameter will result in the entire endpoint being disabled while disabling an "optional" parameter will cause an error to be returned only if the parameter is provided.

### Viewing the Current Configuration

The Indexer has a default set of disabled parameters.  To view the disabled parameters issue:
```
~$ algorand-indexer api-config
```

This will output ONLY the disabled parameters in a YAML configuration.  To view all parameters (both enabled and disabled) issue:

```
~$ algorand-indexer api-config all
```

### Interpreting The Configuration

> Below is a sample output of what the output of `algorand-indexer api-config` will look like:

Reviewer: ```suggestion
Below is a snippet of the output from `algorand-indexer api-config`:
```
